.github/workflows/dist.yml

```diff
@@ -1,15 +1,14 @@
 name: Publish
 
 on:
+  workflow_dispatch:
   push:
     branches:
       - master
 
 jobs:
   publish:
     name: Publish
-    # Check if the PR is not from a fork
```

Comment:
This github action commits to the repository. Allowing contributors to run this action might lead to security issues.

It does not feel safe to me.

Reply:
Hi @samber It can only run on your own repo, it wouldn't run on your 'samber' repo for example. It just makes it easier for contributors to make sure the action runs correctly on their own repo before submitting a patch.